translations/ui-data-import/en.json

```diff
@@ -108,5 +110,6 @@
   "validation.enterValue": "Please enter a value",
   "validation.fileExtension": "Please enter a valid file extension (e.g. .marc)",
   "validation.fileExtension.extension.invalid": "New record not created: File extension {value} is not a valid format",
-  "validation.fileExtension.duplication.invalid": "New record not created: File extension {value} already exists"
+  "validation.fileExtension.duplication.invalid": "New record not created: File extension {value} already exists",
```

Comment:
I thought about improving translations in order to reduce copy paste. I see that this translation string is the same as next one apart from entity type: 'File extension' versus 'Job Profile'. Please scan for similar translation strings and make them reusable.